builtin/providers/google/resource_container_cluster.go

```diff
@@ -146,7 +146,51 @@ func resourceContainerCluster() *schema.Resource {
 				Default:  "default",
 				ForceNew: true,
 			},
-
+			"subnetwork": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"addons_config": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
```

Comment:
Why `Maxitems: 1`? When I try:

```
        addons_config: {
          http_load_balancing: {
            disabled: false
          },
          horizontal_pod_autoscaling: {
            disabled: false
          }
        },
```

it errors: `addons_config: attribute supports 1 item maximum, config has 2 declared`